src/lib/cadastre/mun/shared/mun.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable, Inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { ApiService } from 'src/lib/core/api';
+import {
+  Mun,
+  MunNom,
+  MunApiConfig,
+  MunNomListResponse,
+} from 'src/lib/cadastre/mun/shared/mun.interfaces';
+
+
+@Injectable({
+  providedIn: 'root'
+})
```

Comment:
You should remove {providedIn: 'root'} because this not actually provided in root. We'll do the same everywhere else it applies.